.circleci/config.yml

```diff
@@ -319,7 +319,53 @@ commands:
             - ~/.cache/go-build
             - ~/.cache/tinygo
             - /go/pkg/mod
-
+  arch-release:
+    steps:
+      - run:
+          name: Install dependencies
+          command: pacman -Sy --noconfirm openssh git pacman-contrib binutils
+      - run:
+          name: Create TinyGo user
+          command: useradd -m tinygo
+      - run:
+          name: Become TinyGo user
+          command: su tinygo
+      - run:
+          name: Start SSH Agent
+          command: eval $(ssh-agent -s)
+      - run:
+          name: Add SSH_PRIVATE_KEY identity
```

Comment:
Can we rename this `ARCH_RELEASE_SSH_PRIVATE_KEY`?

Reply:
Done!